.circleci/config.yml

```diff
@@ -6,6 +6,18 @@ common: &common
   working_directory: ~/repo
   steps:
     - checkout
+    - run:
+        name: merge pull request base
+        command: |
+          if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then
+            PR_INFO_URL=https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$CIRCLE_PR_NUMBER
+            curl -L "$PR_INFO_URL" > /tmp/pr-info.json
+
+            PR_BASE_BRANCH=$(python -c 'import json, sys; obj = json.loads(open("/tmp/pr-info.json").read()); sys.stdout.write(obj["base"]["ref"])')
+            git fetch origin "+${PR_BASE_BRANCH}:circleci/pr-base"
+
+            git merge --no-edit circleci/pr-base
```

Comment:
What happens on a failure to merge here? I guess `git merge` would return an error code. Would the error code from the command cause the test to fail?

Reply:
Yes, I think the command would return non-zero exit status and cause the build job to fail.  I'll check that specifically and confirm.